### PR TITLE
Updated README.md to mention build_runner based commands

### PR DESCRIPTION
**Pull request by eernstg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,9 +140,47 @@ dependencies:
 You may also wish to specify constraints on the version, depending on the
 approach to version management that your software otherwise employs.
 
-In order to generate code, you will need to write a tiny Dart program which
-imports the actual builder. We'll assume that you store this program as
-`tool/builder.dart` as seen from the root of your package. Here's the code:
+In order to generate code you will need to run the following command from the
+root directory of your package:
+
+```shell
```
Comment (chalin) [inline README.md]: FYI, using `console` instead of `shell` (here and below) should give you slightly better syntax highlighting.

File: README.md [hunk 2]
```diff
@@ -140,9 +140,47 @@ dependencies:
...
+something that changed. This is the recommended approach.
+
+However, if you have a complex setup then you may wish to study the packages
+`build`, `build_runner`, etc. in order to learn more about how to control
+the build process in more detail.
+
+To give a hint about a more low-level approach, you may choose to write a tiny
+Dart program which imports the actual builder, and run the build process from
+there, rather than using `pub run build_runner ...`. This will give you more
+control, but also more ways to shoot yourself in the foot.
```
Comment (chalin) [inline README.md]: lol

File: README.md (hunk 1 above)
Comment (eernstg) [reply README.md]: Thanks!

File: README.md (hunk 2 above)
Comment (eernstg) [reply README.md]: Should have written feet. ;-)

**Pull request merged**